### DISCUSSION TITLE
How to change telescope theme

### DISCUSSION THREAD

**Discussion by emnnipal** [LazyVim/LazyVim#1127] [Q&A] [answered]
Hi, I'm very new to neovim and mostly use neovim distributions like nvchad and lunarvim. I just want to ask, how can I change the default telescope theme to a dropdown? I checked the Telescope documentation, but I'm having a hard time overriding the telescope options here in LazyVim. Here's my configuration below

```
  {
    "nvim-telescope/telescope.nvim",
    dependencies = {
      "nvim-telescope/telescope-fzf-native.nvim",
      build = "make",
      config = function()
        require("telescope").load_extension("fzf")
      end,
    },
    opts = {
      defaults = {
        theme = "dropdown",
      },
      pickers = {
        find_files = {
          theme = "dropdown",
        },
      },
    },
  }
```

**Comment by jyuan0** [comment]
The config you have should change the theme for the find_files picker, but I think you have to set it for each picker because `theme` isn't a valid key for `defaults`.

**Reply by jyuan0** [reply, marked as answer]
Instead of setting the theme for each picker, you can copy the theme [settings](https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/themes.lua#L27) to `defaults`:
```
        defaults = {
          results_title = false,
          sorting_strategy = "ascending",
          layout_strategy = "center",
          layout_config = {
            preview_cutoff = 1, -- Preview should always show (unless previewer = false)
            width = function(_, max_columns, _)
              return math.min(max_columns, 80)
            end,
            height = function(_, _, max_lines)
              return math.min(max_lines, 15)
            end,
          },
          border = true,
          borderchars = {
            prompt = { "─", "│", " ", "│", "╭", "╮", "│", "│" },
            results = { "─", "│", "─", "│", "├", "┤", "╯", "╰" },
            preview = { "─", "│", "─", "│", "╭", "╮", "╯", "╰" },
          },
        }
```

You can also get these values dynamically and set the table, rather than copying.

**Reply by emnnipal** [reply]
This works even better than what I imagined and it is also highly customizable. Thank you very much!
